goodmod.yaml

```diff
@@ -0,0 +1,25 @@
+# goodmod.yaml
+# This file is tracking branches and tags used in go.mod file replaces
+# Please follow https://github.com/mfojtik/goodmod#installation to get the tool.
+# Run `$ goodmod replace --apply` to apply these rules on `go.mod`
+#
+rules:
+  - paths:
+    - k8s.io/*
+    excludes:
+    - k8s.io/klog
+    - k8s.io/utils
+    tag: kubernetes-1.16.2
+
+  - paths:
+    - github.com/kubernetes-sigs/kube-storage-version-migrator
+    commit: bdca3bf7d454105b7853d0d60b385877486a9dc6
+
+  - paths:
+    - github.com/openshift/*
```

Comment:
I would rather like to see these written out, not with a wildcard.

Reply:
fixed.